### ISSUE TITLE
Modal GPU Integration for Coordizer Harvesting

### ISSUE THREAD

**Issue by GaryOcean428** [GaryOcean428/vex-agent#22]
## Summary

CoordizerV2 (now on `feat/coordizer-v2-resonance-bank`) needs GPU compute for the harvesting phase. Railway doesn't provide GPUs. Modal provides serverless A10G GPUs at ~$0.000306/sec (~$0.16 per 10K samples).

## Architecture Decision

**Option A: Modal as webhook/API** (chosen)

Railway triggers → HTTP POST → Modal GPU endpoint → capture logits → return to Railway → transform to basin coords via CoordizerV2.

This keeps Railway as the control plane and Modal as a stateless GPU worker.

### Phase 1: Modal Function (`vex_coordizer_harvest_modal.py`)

```python
@app.cls(gpu="A10G", timeout=600, container_idle_timeout=300)
class CoordizerHarvester:
    @modal.enter()
    def load_model(self):
        # Load LiquidAI/LFM2.5-1.2B-Thinking to GPU
        
    @modal.fastapi_endpoint(requires_proxy_auth=True)
    async def harvest(self, request: HarvestRequest):
        # Run forward pass, capture full softmax distribution
        # Return: {success, vocab_size, tokens, logits, elapsed_seconds}
```

Security: Modal Proxy Auth Tokens (`wk-*/ws-*` headers)

### Phase 2: Railway Integration

- Add `ModalConfig` to `kernel/config/settings.py` (enabled, harvest_url, token_id, token_secret, timeout)
- Replace `_harvest_phase()` in `kernel/coordizer/gpu_harvest.py` with async httpx call to Modal endpoint
- Transform logits → basin coords via `coordizer_v2.compress`
- Fallback to synthetic data if Modal unavailable

### Phase 3: One-Time Setup

- `@app.function` to download model to Modal Volume (cached across invocations)
- Volume: `vex-models` — persists model weights between cold starts

## Cost Estimate

| Metric | Value |
|--------|-------|
| A10G rate | $0.000306/sec |
| Per 10K samples | ~$0.16 |
| Daily harvest (50K) | ~$0.80 |
| Monthly | <$5 |

## Acceptance Criteria

- [ ] Modal function deploys and responds to authenticated requests
- [ ] Railway can trigger harvest via HTTP POST
- [ ] Logits are correctly transformed to Δ⁶³ basin coordinates
- [ ] Fallback to synthetic works when Modal is unavailable
- [ ] Auth tokens are stored as Railway environment variables, not in code

## References

- CoordizerV2 source: `kernel/coordizer_v2/` on `feat/coordizer-v2-resonance-bank`
- Harvest module: `kernel/coordizer_v2/harvest.py`
- Compress module: `kernel/coordizer_v2/compress.py`
